public/js/webcam.js

```diff
@@ -59,10 +72,22 @@ function postTracks(dataURL) {
     return axios(config)
         .then(() => {
             console.log("SUCCESS - put tracks, now taking to tracks");
-            location.href = "/tracks";
+            location.href = '/tracks';
```

Comment:
How come single quotation?

Reply:
Hmm I don't remember changing this but don't think I had a particular reason, I can check it back?

Reply:
Sure yah it doesnt really matter tbh!